### ISSUE TITLE
cli.test_user.UserWithCleanUpTestCase.test_positive_update_all_roles not isolated

### ISSUE THREAD

**Issue by rplevka** [SatelliteQE/robottelo#6153]
besides that i don't really know what this test is supposed to really test (it look quite useless to me),
it also lacks the isolation:

It first lists all the roles and then creates an user and assigns the roles to it one by one.
- However, some other role-related test runs in parralel, creating and managing its own custom role, including deleting it at the end.
This causes the first test to discover this role during initial listing and sometimes assigning, however it fails to find it at the end during the assertion listing, since it no longer exists

**Comment by pgagne** [comment]
@rplevka any reason we can't just flag this to not run in parallel?

**Comment by rplevka** [comment]
@pgagne - putting stuff to run in a single thread is pretty much a last-resort solution - I would prefer rewriting the test to be isolated if it is possible - and in this case it certainly is.
I also opened up the issue to actually question the purpose of the test - what is it supposed to test? that the roles are really assigned to the user? okay - but why all of them? i'm struggling to get the point of this test.

**Comment by rplevka** [comment]
@pgagne, what do you think? Shall we remove it?

**Comment by sghai** [comment]
@rplevka I see the case we are trying to assign all roles to a user  except the default one. However I see there are  two more tests where we are updating user with one role and then with multiple roles.

So I feel the test where we are assigning `multiple` roles to user is similar to the one where we are assigning `all` roles.

**Comment by sghai** [comment]
@SatelliteQE/quality-engineers any one has any thought around this test case ?

**Comment by JacobCallahan** [comment]
I definitely agree with @sghai, this test seems unnecessary since the functionality is already covered.
